### PR TITLE
changes made to save Switzerland's states and also for access logs fi…

### PR DESCRIPTION
**Pull request by kgaurav1**
changes made to save Switzerland's states and also for access logs file directory

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -47,6 +47,6 @@ management.security.enabled=false
 #=================================
 # access-logs
 #=================================
-server.tomcat.basedir=/logs
+server.tomcat.basedir=.
```
Comment (charanm) [inline src/main/resources/application.properties]: Is this change necessary? Wouldn't this change the log locations?

Comment (kgaurav1) [reply src/main/resources/application.properties]: Yes, It would change access logs location to logs folder which as of now is /logs/logs

Comment (charanm) [reply src/main/resources/application.properties]: Ok fine !!
